src/scripts.js

```diff
@@ -302,12 +302,20 @@ function pressEnterSearch(event) {
 
 function searchRecipes() {
   showAllRecipes();
-  let searchedRecipes = recipeData.filter(recipe => {
+  let searchedRecipes = recipes.filter(recipe => {
     return recipe.name.toLowerCase().includes(searchInput.value.toLowerCase());
-  });
+  }); 
   filterNonSearched(createRecipeObject(searchedRecipes));
 }
 
+// function searchIngredients() {
+//   showAllRecipes();
+//   let ingredientNames = recipes.forEach(recipe => {
+//     return recipe.ingredients.filter(ingredient => {
+//       return ingredient.toLowerCase().includes(searchInput.value.toLowerCase());
+//   });
+// }
+
```

Comment:
Yeah this might need to be refactored with some of the tools I added (sorry!) We can merge and then get it all cleaned up.